src/SprykerEco/Zed/UnzerApi/Persistence/Propel/Schema/spy_unzer_api.schema.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<database xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed" xsi:noNamespaceSchemaLocation="http://static.spryker.com/schema-01.xsd" namespace="Orm\Zed\UnzerApi\Persistence" package="src.Orm.Zed.UnzerApi.Persistence">
+
+    <table name="spy_payment_unzer_api_log" idMethod="native" allowPkInsert="true" isCrossRef="true">
+        <column name="id_payment_unzer_api_log" required="true" type="INTEGER" primaryKey="true" autoIncrement="true"/>
+        <column name="request_type" required="true" type="VARCHAR" size="255"/>
+        <column name="url" required="true" type="VARCHAR" size="255"/>
+        <column name="request" required="true" type="LONGVARCHAR"/>
+        <column name="response" required="true" type="LONGVARCHAR"/>
+        <column name="is_success" required="true" type="BOOLEAN"/>
```

Comment:
Please, use `is_successful` instead of `is_success`.

Reply:
Fixed